github_delete.py

```diff
@@ -0,0 +1,68 @@
+# github_delete_all.py
+# Usage:  python3 github_delete.py --prefix SOMEPREFIX
+# Requires: gh CLI
+# Modified by See-Mong Tan <see-mong.tan@wwu.edu>
+# Dan Wallach <dwallach@rice.edu>
+# Available subject to the Apache 2.0 License
+# https://www.apache.org/licenses/LICENSE-2.0
+
+import argparse
+import subprocess
+from github_config import *
+from github_scanner import *
+
+parser = argparse.ArgumentParser(description='Clone matching GitHub repos all at once.')
+parser.add_argument('--token',
+                    nargs=1,
+                    default=[default_github_token],
+                    help='GitHub API token')
+parser.add_argument('--org',
+                    nargs=1,
+                    default=[default_github_organization],
+                    help='GitHub organization to scan, default: ' + default_github_organization)
+parser.add_argument('--prefix',
+                    nargs=1,
+                    default=[default_prefix],
+                    help='Prefix on projects to match (default: match all projects)')
+parser.add_argument('--safe',
+                    action="store_true",
+                    default=False,
+                    help="Creates cloned repos without the API token embedded. Pushing may not work," +
+                         " but repos are safer to share (default: API token is embedded)")
+parser.add_argument('--out',
+                    nargs=1,
+                    default=["."],
+                    help='Destination directory for GitHub clones (default: current directory)')
```

Comment:
You're not using these arguments, so don't ask for them.

Reply:
But you should keep the `--safe` argument around and define it to just tell you the list of repos that are about to deleted, except taking no action. Deleting repos is a somewhat dangerous thing, so having a feature to list what's about to get nuked before you nuke it? That seems wise.